modules/plugins/statusline/feline/config.nix

```
@@ -0,0 +1,21 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+
+  cfg = config.vim.statusline.feline-nvim;
+in {
+  config = mkIf cfg.enable {
+    vim = {
+      lazy.plugins.feline-nvim = {
+        event = "UIEnter";
```

```suggestion
        event = "DeferredUIEnter";
```
This is debatable, but if the goal is to start accepting user input as soon as possible, then `DeferredUIEnter` should be used

---

Is there a benefit to taking user input as soon as possible?

---

I mean yeah? You wouldn't want a 10s startup time would you